docs/apis/table.md

```diff
@@ -57,6 +52,170 @@ The following dependency must be added to your project in order to use the Table
 
 Note that the Table API is currently not part of the binary distribution. See linking with it for cluster execution [here]({{ site.baseurl }}/apis/cluster_execution.html#linking-with-modules-not-contained-in-the-binary-distribution).
 
+
+Registering and Accessing Tables
```

Comment:
No, I mean we should move the subsection about accessing tables to the respective sections and rename this section to "Registering Tables".